src/azure-cli/azure/cli/command_modules/feedback/tests/latest/test_feedback.py

```diff
@@ -29,6 +29,7 @@
 logger = logging.getLogger(__name__)
 
 
+@unittest.skip('alias extension is not working: https://github.com/Azure/azure-cli/issues/29422')
```

Comment:
After removing `markupsafe`, the `alias` extension does not work anymore. Previously, the CLI's `markupsafe` was loaded; now it loads the outdated alias's `markupsafe` package.
```
az alias create -n foo123 -c bar123
Unable to load extension 'alias: cannot import name 'soft_unicode' from 'markupsafe' (C:\Users\kk\.azure\cliextensions\alias\markupsafe\__init__.py)'. Use --debug for more information.
```